.circleci/config.yml

```diff
@@ -133,6 +133,23 @@ jobs:
       - run: yarn lint
       - run: yarn test
 
+  docs:
+    docker:
+      - image: circleci/python:latest
+    steps:
+      - checkout
+      - setup_remote_docker
+      - run:
+          name: Install dependencies
+          command: |
+            cd docs
```

Comment:
you should set the working directory at the top of the job config, like in `example_ens`

Reply:
not yet done